### PR TITLE
Blog: Cordova's Timeline for Node 6.x & 8.x Deprecation & Node 12.x Support

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected
none

### Motivation and Context
Old node deprecation and new version release.

### Description
Blog post about Cordova's timeline with node version support.

### Testing
none

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: www/_posts/2019-03-24-nodejs-6.x-8.x-deprecation-timeline.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+author:
+    name: Bryan Ellis
+title: "Node 6.x and 8.x Deprecation Timeline for Apache Cordova"
+categories: news
+tags: deprecation timeline node
+---
+
+Currently, all of the latest released platforms, tools, and core libraries require a minimum version of `node 6.x` to be installed. Additionally, we have added support for `node 10.x`.
+
+The Node.js Foundation has reported that the End-of-Life (EOL) for version 6.x will be April 30th, 2019. Looking ahead, the Node.JS Foundation has also scheduled the deprecation of 8.x on December 31, 2019 to be inaligned with the scheduled End-of-Life of OpenSSL-1.0.2.
```
Comment (dpogue) [inline www/_posts/2019-03-24-nodejs-6.x-8.x-deprecation-timeline.md]: "inaligned" isn't a word. Did you mean "aligned" or "in alignment"?

**Comment by janpio** [comment]
Don't forget to rename the file before publish as it implies the publication date.

File: www/_posts/2019-03-24-nodejs-6.x-8.x-deprecation-timeline.md (hunk 1 above)
Comment (erisu) [reply www/_posts/2019-03-24-nodejs-6.x-8.x-deprecation-timeline.md]: Yes, `aligned` was what I meant to write.

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* All AppVeyor/Travis CI builds will test on Node.js 6.x, 8.x, 10.x, and 12.x.
+* Update requirement check and notify users that using Node.js 6.x, was deprecated by the Node team and is not recommended to use and should upgrade to the minimum LTS (10.x).
+
+### **September 2019**
+
+* Update requirement check and notify deprecation warning if the user is using Node.js 6.x and 8.x and recommend to upgrade to the minimum LTS (10.x).
+
+### **December 31st, 2019**
+
+* Deprecate use of Node.js 6.x
```
Comment (GedasGa) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
* Deprecated use of Node.js 6.x
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* Deprecate use of Node.js 8.x
```
Comment (GedasGa) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
* Deprecated use of Node.js 8.x
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md (hunk 2 above)
Comment (purplecabbage) [reply www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]: We should deprecate now, and remove support at the end of the year, or sooner for 6.

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+<!--more-->
+
+## Cordova Timeline
+
+### **March 2019**
+
+* All AppVeyor/Travis CI builds will continue to test on Node.js 6.x, 8.x, and 10.x.
+
+### **April 30th, 2019**
```
Comment (erisu) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
### **April 2019**
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+* All AppVeyor/Travis CI builds will test on Node.js 6.x, 8.x, 10.x, and 12.x.
+* Update requirement check and notify users that using Node.js 6.x, was deprecated by the Node team and is not recommended to use and should upgrade to the minimum LTS (10.x).
```
Comment (erisu) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
* Update requirement check and notify users: `Warning: Node.js 6.x has been deprecated. Please upgrade to the latest Node.js version available (LTS version recommended).`
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### **September 2019**
+
+* Update requirement check and notify deprecation warning if the user is using Node.js 6.x and 8.x and recommend to upgrade to the minimum LTS (10.x).
+
+### **December 31st, 2019**
+
+* Deprecate use of Node.js 6.x
+* Deprecate use of Node.js 8.x
+* Officially Add support for Node.js 12.x
```
Comment (erisu) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
* Added Node.js 12.x  support.
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+### **April 30th, 2019**
+
+* All AppVeyor/Travis CI builds will test on Node.js 6.x, 8.x, 10.x, and 12.x.
+* Update requirement check and notify users that using Node.js 6.x, was deprecated by the Node team and is not recommended to use and should upgrade to the minimum LTS (10.x).
+
+### **September 2019**
+
+* Update requirement check and notify deprecation warning if the user is using Node.js 6.x and 8.x and recommend to upgrade to the minimum LTS (10.x).
+
+### **December 31st, 2019**
```
Comment (erisu) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
### **December 2019 / January 2020**
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+* All AppVeyor/Travis CI builds will continue to test on Node.js 6.x, 8.x, and 10.x.
+
+### **April 30th, 2019**
+
+* All AppVeyor/Travis CI builds will test on Node.js 6.x, 8.x, 10.x, and 12.x.
+* Update requirement check and notify users that using Node.js 6.x, was deprecated by the Node team and is not recommended to use and should upgrade to the minimum LTS (10.x).
+
+### **September 2019**
+
+* Update requirement check and notify deprecation warning if the user is using Node.js 6.x and 8.x and recommend to upgrade to the minimum LTS (10.x).
```
Comment (erisu) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
* Update requirement check and notify users: `Warning: Node.js 8.x has been deprecated. Please upgrade to the latest Node.js version available (LTS version recommended).`
```

File: www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+### **March 2019**
+
+* All AppVeyor/Travis CI builds will continue to test on Node.js 6.x, 8.x, and 10.x.
+
+### **April 30th, 2019**
+
+* All AppVeyor/Travis CI builds will test on Node.js 6.x, 8.x, 10.x, and 12.x.
+* Update requirement check and notify users that using Node.js 6.x, was deprecated by the Node team and is not recommended to use and should upgrade to the minimum LTS (10.x).
+
+### **September 2019**
```
Comment (erisu) [inline www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md]:
```suggestion
### **Fall 2019**
```

**Comment by purplecabbage** [comment]
+1
Thanks Erisu!

@purplecabbage
risingj.com

**Pull request merged**